src/app/selectors.js

```diff
@@ -105,6 +105,92 @@ const getModelStatusByUUID = (modelUUID, modelStatuses) => {
   return null;
 };
 
+// Highest status to the right
+const statusOrder = ["running", "alert", "blocked"];
+
+/**
+  Returns a grouped collection of model statuses.
+  @param {Object} modelStatuses
+  @returns {Function} The grouped model statuses.
+*/
+const groupModelStatuses = modelStatuses => {
```

Comment:
The logic used here is pretty basic. In the follow-up I'll expand it with additional entities to check to determine the model status. Note the empty `getUnitStatusGroup` function below.